src/app/models/timeslot.rb

```diff
@@ -25,7 +25,7 @@ def initialize(start, stop)
     end
 
     def to_s(with_day: false)
-      "#{start_day + ' ' if with_day}#{start_time} - #{end_time}"
+      "#{start_day + ' ' if with_day}#{start_time} – #{end_time}"
```

Comment:
Ahh, yes, I didn't read the end of your comment -- just hex. I'm pretty sure there isn't a clear way to communicate it is an en dash unless you assigned it to a local var and interpolating it. 

…or an atomic commit. 👍